### DISCUSSION TITLE
ERROR 200: one or more errors in input file. How to resolve this error message from pyswmm?

### DISCUSSION THREAD

**Discussion by nnet55** [pyswmm/pyswmm#434] [Q&A] [answered]
Hi
I am trying to run a model in pyswmm and getting error message "ERROR 200: one or more errors in input file.". This model is running perfectly ( 1 warning)  in SWMM while when running it on pyswmm its not working.

(![image](https://user-images.githubusercontent.com/32679410/235368744-13c11dda-cbef-4ad3-85ca-d3a899f25e1a.png)

[net_try_2.zip](https://github.com/OpenWaterAnalytics/pyswmm/files/11361991/net_try_2.zip)

**Comment by bemcdonnell** [comment]
@nnet55 what version of pyswmm? And what version of EPA SWMM does it run with?

**Comment by bemcdonnell** [comment, marked as answer]
@nnet55 if you open the RPT file it might tell you what the errors are, specifically.
